components/Layout/Footer/Footer.tsx

```diff
@@ -0,0 +1,26 @@
+import { useCompanyInformation } from '@/hooks/useCompanyInformation';
+
+const Footer = () => {
+    const companyInformation = useCompanyInformation();
+    if (!companyInformation) {
+        return null;
+    }
+    return (
+        <footer>
+            <h2>
+                <>About</>
```

Comment:
Redundant fragment?

Reply:
it is, but for some reason prettier removes the space. I could add the space inside the other `{}` or create a span, but I think a fragment is more readable. What do you think?

![Kapture 2021-04-20 at 15 55 02](https://user-images.githubusercontent.com/4324982/115407997-cb4d5d00-a1f0-11eb-90f3-2bcb46ac8cd9.gif)

Reply:
Doesn't matter 🙂 Just leave it as is